### PR TITLE
perf: improve performance mongodb and redis

### PR DESCRIPTION
**Pull request by chrisleekr**
## Description

- Added indexes for MongoDB
- Updated configuration to use cache as well to improve performance
- Updated redis.conf to disable AOF for improving performance
- Moved grid/manual orders to MongoDB from Redis since Redis is not persistent anymore
- Updated MongoDB to not acknowledge insert/update/delete

## Related Issue

#262

## Motivation and Context

Since the bot was showing performance issues in Raspberry Pi, I started to look at how to improve the performance.

By disabling AOF in Redis and write concern in MongoDB(risky but worth), it starts showing lower CPU usage.

## Screenshots (if appropriate):

image

### CODE REVIEW

File: migrations/1629229061520-create-collection-index.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  const logger = rootLogger.child({
+    gitHash: process.env.GIT_HASH || 'unspecified',
+    migration: path.basename(__filename)
+  });
+
+  const database = await mongo.connect(logger);
+
+  logger.info('Start migration');
+
+  let collection;
```
Comment (habibalkhabbaz) [inline migrations/1629229061520-create-collection-index.js:14]: Using variables from outside `Promise` causing incorrect indexes mapping to collections.
For example, I see the `trailing-trade-symbols-key-idx` added in the **trailing-trade-common** collection.
I don't know if that was the main reason but when I move the variables inside the `Promise`, it solves the issue.

**Review by habibalkhabbaz — CHANGES_REQUESTED** [review]
Thanks @chrisleekr for this nice improvements 🏆 
I tested this pull request, and I only found one issue.

File: migrations/1629229061520-create-collection-index.js (hunk 1 above)
Comment (chrisleekr) [reply migrations/1629229061520-create-collection-index.js:14]: Oh really? Interesting. Let me test it out.

Comment (chrisleekr) [reply migrations/1629229061520-create-collection-index.js:14]: You are right! Gee, how did I miss this. Let me fix it up.

**Pull request merged**